<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookRecords</class>
 <widget class="QWidget" name="BookRecords">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Book Records</string>
  </property>

  <layout class="QVBoxLayout" name="verticalLayout_main">

   <!-- HEADER -->
   <item>
    <layout class="QHBoxLayout" name="layout_header">
     <item>
      <widget class="QLabel" name="label_header">
       <property name="text">
        <string>Book Records</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 22px; font-weight: bold;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer_header" >
       <property name="orientation"><enum>Qt::Horizontal</enum></property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>


   <!-- SEARCH AREA -->
   <item>
    <layout class="QHBoxLayout" name="layout_search">

     <item>
      <widget class="QComboBox" name="comboSearchType">
       <item><property name="text"><string>Title</string></property></item>
       <item><property name="text"><string>Author</string></property></item>
       <item><property name="text"><string>Category</string></property></item>
      </widget>
     </item>

     <item>
      <widget class="QLineEdit" name="lineSearch">
       <property name="placeholderText">
        <string>Enter search text...</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="btnSearch">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="btnClearSearch">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>

    </layout>
   </item>


   <!-- BOOKS TABLE -->
   <item>
    <widget class="QTableWidget" name="tableBooks">
     <column>
      <property name="text"><string>ID</string></property>
     </column>
     <column>
      <property name="text"><string>Title</string></property>
     </column>
     <column>
      <property name="text"><string>Author</string></property>
     </column>
     <column>
      <property name="text"><string>Category</string></property>
     </column>
     <column>
      <property name="text"><string>Copies</string></property>
     </column>
     <column>
      <property name="text"><string>Status</string></property>
     </column>
    </widget>
   </item>


   <!-- ACTION BUTTONS -->
   <item>
    <layout class="QHBoxLayout" name="layout_actions">

     <item>
      <widget class="QPushButton" name="btnAddBook">
       <property name="text"><string>Add Book</string></property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="btnEditBook">
       <property name="text"><string>Edit Book</string></property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="btnDeleteBook">
       <property name="text"><string>Delete Book</string></property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="btnLowStock">
       <property name="text"><string>Low Stock</string></property>
      </widget>
     </item>

     <item>
      <spacer name="spacer_actions">
       <property name="orientation"><enum>Qt::Horizontal</enum></property>
      </spacer>
     </item>

    </layout>
   </item>

  </layout>

 </widget>
 <resources/>
 <connections/>
</ui>
